src/main/java/com/synopsys/integration/alert/web/controller/UIComponentController.java

```diff
@@ -103,7 +103,7 @@ public UIComponent getDistributionUIComponent(@RequestParam(value = "providerNam
             final UIComponent providerUIComponent = providerUIConfig.generateUIComponent();
             final List<ConfigField> combinedFields = new ArrayList<>();
             final ConfigField name = new TextInputConfigField("name", "Name", true, false);
-            final ConfigField frequency = new SelectConfigField("frequency", "Digest type", true, false, Arrays.stream(DigestType.values()).map(type -> type.getDisplayName()).collect(Collectors.toList()));
+            final ConfigField frequency = new SelectConfigField("frequency", "Digest type", true, false, Arrays.stream(FrequencyType.values()).map(type -> type.getDisplayName()).collect(Collectors.toList()));
```

Comment:
Should Digest type here be changed to Frequency?

Reply:
yes it should